tests/common/fixtures/ptfhost_utils.py

```diff
@@ -197,49 +198,54 @@ def run_icmp_responder(duthost, ptfhost, tbinfo):
 
     yield
 
-    logging.info("Stop running icmp_responder")
-    ptfhost.shell("supervisorctl stop icmp_responder")
+    if tbinfo['topo']['type'] == 't0':
```

Comment:
Is this required? The check at the beginning of this fixture yields and returns if testbed type is not `t0`.